src/main/groovy/edu/oregonstate/mist/catalogapitest/core/Course.groovy

```diff
@@ -0,0 +1,99 @@
+package edu.oregonstate.mist.catalogapitest.core
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+import javax.xml.bind.annotation.XmlRootElement
+
+@XmlRootElement
+class Course {
+
+    private Integer cid
+    private Integer crn
+    private String courseName
+    private String instructor
+    private String day
+    private String time
+    private String location
+
+    public Course() {
+        // Jackson Deserialization
+    }
+
+    public Course(Integer cid, Integer crn, String courseName, String instructor, String day, String time, String location) {
```

Comment:
suggestion: groovy also provides implicit constructors, so see if you can simplify or remove this.